AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -320,6 +320,7 @@ scootertechno.com,mysize.ch,aeotec.com,geschichte-wissen.de,4kfilme.de,pk.edu.pl
 !
 ! NOTE: Regular rules
 !
+baomoi.com##.bottom-0:has(a[href="/staticpages/privacy.epi"])
```

Comment:
Can be used this rule
https://github.com/AdguardTeam/AdguardFilters/blob/20bfb30847a6e15997dac1bd50871723f51df146/AnnoyancesFilter/Cookies/sections/cookies_specific.txt#L152
it seems it does not break anything in such cases.